src/test/java/JUnitTest.java

```diff
@@ -0,0 +1,160 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import org.junit.jupiter.api.Test;
+import java.util.List;
+
+public class JUnitTest {
+
+    @Test
+    public void testCategoryCreation() {
```

Comment:
We have this naming convention for the test: methodToTest_inputOfTest_expectedOutput

Same problems found across the file